app/sections/view-capabilities/macro.njk

```diff
@@ -0,0 +1,32 @@
+{% from 'components/view-epics/macro.njk' import viewEpics %}
+{% from 'components/view-section-heading/macro.njk' import viewSectionHeading %}
+{% from 'components/view-expandable-section/macro.njk' import viewExpandableSection %}
+{% from 'components/view-data-text/macro.njk' import viewDataText %}
+{% from 'components/view-data-link/macro.njk' import viewDataLink %}
+
+{% macro viewCapabilities(params) %}
+  {% if params %}
```

Comment:
we probably don't want to show this entire section when params.capabilities.length is false